qiskit/providers/fake_provider/fake_backend.py

```diff
@@ -344,98 +349,23 @@ def run(self, run_input, **options):
         if pulse_job:  # pulse job
             raise QiskitError("Pulse simulation is currently not supported for V2 fake backends.")
         # circuit job
-        if not _optionals.HAS_AER:
+
+        if _optionals.HAS_AER:
+            from pkg_resources import parse_version
+            from qiskit_aer import version as aer_version
```

Comment:
This cannot succeed if `Aer` is <0.11, because it didn't publish itself with this name until Aer 0.11.0.